<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="single_radio">
       <property name="text">
        <string>Single Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="playlist_radio">
       <property name="text">
        <string>Playlist</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="audio_checkbox">
       <property name="text">
        <string>Audio only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="open_dialog">
       <property name="text">
        <string>Select Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filepath_input">
       <property name="cursor">
        <cursorShape>BusyCursor</cursorShape>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="link_input">
       <property name="placeholderText">
        <string>www.enteryourlinkhere.com</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_download">
       <property name="text">
        <string>Start Download</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
